src/mappings/core.ts

```diff
@@ -576,4 +604,15 @@ export function handleSwap(event: Swap): void {
     amount1Total.times(token1.derivedNativeCurrency as BigDecimal).times(bundle.nativeCurrencyPrice)
   )
   token1DayData.save()
+
+  // update unique day swaps
+  let uniqueDailyInteractionsData = updateUniqueDailyInteractions(event)
+
+  // increment unique daily swaps
+  if (!uniqueDailyInteractionsData.addresses.includes(swap.to as Bytes)) {
```

Comment:
`swap.to` might be different than the original sender. For example, when I want to swap but receive the funds to another wallet :) Should check for that.

Reply:
So we should check for the `from` instead of the `to` I guess

Reply:
That is correct!